types/relay-runtime/relay-runtime-tests.tsx

```diff
@@ -263,17 +263,17 @@ query FooQuery {
 const node: ConcreteRequest = (function () {
     var v0 = [
         {
-            kind: 'ScalarField',
+            kind: 'ScalarField' as const,
```

Comment:
we cannot do this, sorry. The relay compiler is responsible for outputting this object, and doesn't current ship an `as const`. 

We may be able too if we also touch the compiler to add this.

Reply:
Thanks, will revert this. Will need to revert the `kind` string literals along with it. :+1:

Reply:
it is a really good thing to add but would require a breaking change on both fronts and a bit of orchestration.

we can make it happen, just not with this PR